From 812a4d13deda25e911b1466ea27a4c55f26c036d Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Tue, 29 Apr 2025 10:24:59 +0300
Subject: [PATCH] [SW-227530] mk: add cmn module for event build

Currently in arc event we dont compile common
module. Added this to use the feature of rate
limiting log, etc...

Change-Id: Id2aa8ed9913dc6712ef92fe620ab8367d18404c5
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/package/event/event.mk b/package/event/event.mk
index 0f1d022..0deaf96 100644
--- a/package/event/event.mk
+++ b/package/event/event.mk
@@ -130,9 +130,14 @@
 	@$(EVENT_CONF_ENV) $(HOST_DIR)/bin/cmake $(EVENT_CONF_OPTS) -S $(EVENT_BUILD_DIR) -B $(EVENT_BUILD_DIR)/build
 endef
 
+define EVENT_COPY_EMB_COMMON_MODS_SRC
+	rsync -au --chmod=u=rwX,go=rX --exclude .svn --exclude .git --exclude .hg --exclude .bzr --exclude CVS $(EMB_COMMON_MOD_SRC_DIR)/ $(@D)/emb_common_mod
+endef
+
 EVENT_POST_RSYNC_HOOKS += HL_VERSION_GEN
 EVENT_POST_RSYNC_HOOKS += EVENT_COPY_ZEPHYR_SRC
 EVENT_POST_RSYNC_HOOKS += EVENT_APP_VERSION_GEN
+EVENT_POST_RSYNC_HOOKS += EVENT_COPY_EMB_COMMON_MODS_SRC
 
 define EVENT_BUILD_CMDS
 	@$(TARGET_MAKE_ENV) $(MAKE) -C $(EVENT_BUILD_DIR)/build all
